weather_mv/loader_pipeline/bq.py

```diff
@@ -143,10 +146,23 @@ def to_table_schema(columns: t.List[t.Tuple[str, str]]) -> t.List[bigquery.Schem
     fields.append(bigquery.SchemaField(DATA_IMPORT_TIME_COLUMN, 'TIMESTAMP', mode='NULLABLE'))
     fields.append(bigquery.SchemaField(DATA_URI_COLUMN, 'STRING', mode='NULLABLE'))
     fields.append(bigquery.SchemaField(DATA_FIRST_STEP, 'TIMESTAMP', mode='NULLABLE'))
+    fields.append(bigquery.SchemaField(S2_LOCATION, 'GEOGRAPHY', mode='NULLABLE'))
 
     return fields
 
 
+def fetch_s2_location(lat, long):
+    """
```

Comment:
As per the [Google Python style guide](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings), the docstring should be formatted a bit differently: 
- The description should be on the same line as the open block quote and end with a period. 
- This looks like it will be a single line docstring
- If this is multiline, please add a short 1-line description on the same line as the open-block quote, and then add a longer description in a paragraph below. 
- No need to say "this function", and "calculates and returns" is a bit redundant.

Specifically, this might look like: 
```
def function_name(args): 
    """Calculates a geography point from an input latitude and longitude.""" 
    # ...
```